llvm/include/llvm/CodeGen/MIRYamlMapping.h

```diff
@@ -631,6 +631,79 @@ LLVM_YAML_IS_SEQUENCE_VECTOR(llvm::yaml::CalledGlobal)
 namespace llvm {
 namespace yaml {
 
+// Struct representing one save/restore point in the
+// 'savePoint' / 'restorePoint' list. One point consist of machine basic block
+// name and list of saved/restored in this basic block registers. There are
+// two forms of Save/Restore point representation:
+// 1. Without explicit register enumeration:
+//      savePoint:       '%bb.n'
+//      restorePoint:    '%bb.n'
+// supported for backward compatibility (in this case we assume that all
+// CalleeSavedRegisters are splilled/restored in these points)
```

Comment:
We ought to make callee saved registers an explicit mask operand on return instructions